crm_rma_location_rma/__openerp__.py

```diff
@@ -0,0 +1,39 @@
+# -*- coding: utf-8 -*-
+##############################################################################
+#
+#    Copyright 2015 Vauxoo
+#    Author: Yanina Aular
+#
+#    This program is free software: you can redistribute it and/or modify
+#    it under the terms of the GNU Affero General Public License as
+#    published by the Free Software Foundation, either version 3 of the
+#    License, or (at your option) any later version.
+#
+#    This program is distributed in the hope that it will be useful,
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU Affero General Public License for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
+
+{
+    'name': 'RMA Location',
+    'version': '8.0.1.0.0',
+    'author': "Vauxoo, Odoo Community Association (OCA)",
+    'license': 'AGPL-3',
+    'website': 'http://www.camptocamp.com,http://www.vauxoo.com',
+    'category': 'Generic Modules/CRM & SRM',
+    'depends': [
+        'stock',
+        'procurement',
```

Comment:
@pedrobaeza No, it doesn't. In fact **```rma```** module is an app and it depends on this module, this is done in this way to make this feature independent like the rest of the modules, and modules like **```crm_claim_rma```** can take advantange of picking types this module create